build.gradle

```diff
@@ -9,6 +9,7 @@ buildscript {
                 details.because 'jcenter redirect'
                 System.out.println ">>>>>>>>> fix grolifant"
             }
+            //TODO is this really needed here?
```

Comment:
this is a safety net which helped with when jcenter was gone. Maybe we can remove it by now, but it also does no harm

Reply:
Yeah I remember and it makes sense I just wondered if it makes sense to go with 0.7.1 instead of force downgrade with this script. Because currently the versions file says 0.7.2

Reply:
yeah, I am also nut sure if we are not already over this...

Reply:
i'll keep that comment, but i will exceed the scope here